modules/serialize/src/identifiers.ts

```diff
@@ -62,8 +62,23 @@ Object.freeze(SequenceIdentifier)
 export enum Maximum {
   // Maximum number of messages which are allowed to be encrypted under a single cached data key
   MESSAGES_PER_KEY = 2 ** 32, // eslint-disable-line no-unused-vars
-  // Maximum number of bytes which are allowed to be encrypted under a single cached data key
-  BYTES_PER_KEY = 2 ** 63 - 1, // eslint-disable-line no-unused-vars
+  /* Maximum number of bytes which are allowed to be encrypted under a single data key
```

Comment:
I want to make sure we're clear in any comments/names/etc that this is the limit applied to cached data key reuse *across messages*, not data key use *within a single message*.

Reply:
pushed update